src/cachingreader.cpp

```diff
@@ -162,66 +159,90 @@ void CachingReader::freeAllChunks() {
     m_allocatedChunks.clear();
     m_mruChunk = NULL;
 
-    QSet<Chunk*> reserved = QSet<Chunk*>::fromList(m_chunksBeingRead.values());
+    for (QVector<Chunk*>::const_iterator it = m_chunks.constBegin();
+         it != m_chunks.constEnd(); ++it) {
```

Comment:
For me these are the regions where the Google style does not apply because of our 4 space indents. 
if () -> 8 indent is required to distinguish 
for () -> 5 indent is IMHO to close to 4. 
Lets say at least 8 and everyone should be happy.